jenkins/ci.suse.de/crowbar.patch

```diff
@@ -0,0 +1,13 @@
+--- a/bin/install-chef-suse.sh 2019-02-01 12:07:26.308101903 +0000
++++ b/bin/install-chef-suse.sh 2019-02-01 12:06:34.868101903 +0000
+@@ -887,8 +887,8 @@
+ 
+ # Use existing SSH authorized keys
+ if [ -f /root/.ssh/authorized_keys ]; then
+-    # remove empty lines and change newline to \n
+-    access_keys=$(sed "/^ *$/d" /root/.ssh/authorized_keys | sed "N;s/\n/\\n/g")
++    # remove empty lines and comments and change newline to \n
++    access_keys=$(sed "/^ *$/d;/^ *#/d" /root/.ssh/authorized_keys | sed "N;s/\n/\\n/g")
+     provisioner_keys=$(json_read "$PROVISIONER_JSON" \
+         attributes.provisioner.access_keys)
+     if [ ! -f "$PROVISIONER_JSON" -o -z "$provisioner_keys" ]
```

Comment:
same here, this could be dropped as you fix it in the Jenkinsfile as well

Reply:
done